fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -142,6 +142,12 @@ public enum OlapTableState {
     @SerializedName(value = "colocateGroup")
     protected String colocateGroup;
 
+    @SerializedName(value = "colocateMv")
+    protected Set<String> colocateMaterializedViewNames = Sets.newHashSet();
```

Comment:
if we have a mv with colocated_mv property set then other mv will also be influenced? am i right?

Reply:
yeap, the current implementation of colocate mv will affect the banlance of other mv